CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+## Before reporting a bug
+* Update to the latest *development version*.
```

Comment:
+1 _(also for the reason that backward compatibility wasn't always there in both directions.. eg. no downgrade without data loss)_